showcase/src/app/router.config.ts

```diff
@@ -18,8 +17,10 @@ function logRegisteredStates(registeredstates: StateDeclaration[]): void {
 
 export function routerConfigFn(router: UIRouter): void {
 	router.trace.enable(Category.TRANSITION);
-	router.plugin(Visualizer);
-
+	// Enable UI-Router visualizer here if needed (for development purposes only)
+	// if (ENV === "development") {
+	// 	router.plugin(Visualizer);  // Visualizer should be imported from "@uirouter/visualizer"
```

Comment:
Why do you disable it always ?

Reply:
Because it is not needed for us in the Showcase, if we need it to check some transitions it will be only locally in our fork while developing ;)

Reply:
That means it should be there in development environment. 
The version we publish is in production mode. That means we could enable it.

Reply:
Nop, because nobody wants to use this in development. If someone wants to use it then he simply needs to uncomment this and that's it, but this should not be commited because it is annoying to have this visualizer all the time in dev

Reply:
Okido, thanks for the explanation